analytical_engine/apps/centrality/katz/katz_centrality.h

```diff
@@ -165,6 +169,14 @@ class KatzCentrality
     }
     ctx.curr_round++;
   }
+
+  bool filterByDegree(const fragment_t& frag, context_t& ctx, vertex_t v) {
+    int degree = frag.GetLocalInDegree(v) + frag.GetLocalOutDegree(v);
```

Comment:
Should we check the direction of fragment ? As I know, if the graph load with `undirected` and `BothOutIn` strategy, the `degree` would double the actual degree of v.

Reply:
Good catch 👍